sdk/os/Env.ooc

```diff
@@ -25,8 +25,13 @@ Env: class {
     set: static func (key, value: String, overwrite: Bool) -> Int {
         if(key != null && value != null) {
             version(windows) {
-                // todo: handle overwrite
-                return putenv( "%s=%s" format(key toCString(), value toCString()) toCString() )
+                exists := false
+                if (!overwrite) {
+                    old := This get(key)
+                    exists = (old != null)
+                }
+                if (overwrite || !exists)
```

Comment:
Or I guess with a garbage collector we can simply do `if (overwrite || This get(key) != null)`?